scripts/utils/bdlClassnameManager

```diff
@@ -29,12 +29,12 @@ const writeFile = promisify(fs.writeFile);
 */
 
 const conversionMap = {
-    // 'btn': 'bdl-Button',
-    // 'btn-group': 'bdl-ButtonGroup',
-    // 'btn-plain': 'bdl-Button--plain',
-    // 'btn-primary': 'bdl-Button--primary',
-    // 'is-disabled': 'bdl-is-disabled',
-    // 'label': 'bdl-Label',
+    btn: 'bdl-Button',
+    'btn-group': 'bdl-ButtonGroup',
+    'btn-plain': 'bdl-Button--plain',
+    'btn-primary': 'bdl-Button--primary',
+    'is-disabled': 'bdl-is-disabled',
```

Comment:
SUIT [naming convention doc](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#components) says <ComponentName> is not optional. `is-disabled` looks more like a modifier and should be used in conjunction with the component itself, e.g. `bdl-SelectButton--is-disabled`.

Reply:
@sahiga `is-disabled` is correct, you can ignore above comment.